packages/paste-website/src/pages/utilities/screen-reader-only/index.mdx

```diff
@@ -0,0 +1,212 @@
+
+Another common use case is for providing context to repetitive actions. Visually the "Buy" button is associated with the phone number via the row highlighting, but to a screen reader user the "Buy" action might be unclear. To provide additional clarification we can add _what_ you would be buying as visually hidden text to the button.
+
+<LivePreview scope={{Table, Thead, Tbody, Tr, Td, Th, Button, Text, ScreenReaderOnly}} language="jsx">
+  {`<Table>
+  <Thead>
+    <Tr>
+      <Th>Number</Th>
+      <Th>Type</Th>
+      <Th>Monthly Fee</Th>
+      <Th>
+        <ScreenReaderOnly>Actions</ScreenReaderOnly>
+      </Th>
+    </Tr>
+  </Thead>
+  <Tbody>
+    <Tr>
+      <Th>
+        <strong>+1 (903) 603-9469</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Mabank, TX
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (903) 603-9469</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (470) 309-0026</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Adairsville, GA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (470) 309-0026</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (484) 662-4088</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Hamburg, PA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (484) 662-4088</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+  </Tbody>
+</Table>`}
+</LivePreview>
+
+## When to use Screen Reader Only
+
+Use the Screen Reader Only component when you are required to provide additional or contextual information to a screen reader user which a sighted user maybe able to infer and when you cannot do so via semantic HTML.
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use to provide additional text content to users of assistive technology"
+    preview={false}
+  />
+  <Dont title="Don't" body="Don't use to hide interactive content" preview={false} />
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/screen-reader-only
+```
+
+#### Usage
+
+```jsx
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+
+<Button variant="secondary" size="small">
+  Buy <ScreenReaderOnly>+1 (484) 662-4088</ScreenReaderOnly>
+</Button>
+```
+
+All the regular HTML attributes (`role`, `aria-*`, `type`, and so on) including the following custom props:
```

Comment:
Should we have recommendations for `aria-describedBy` and screen reader only?

Reply:
I'm not sure I follow. What kind of recommendations are you thinking of?

Reply:
Mostly when you should use aria-described by or the screen reader component, I can think of a few use cases for either but stating the do it like this would be really helpful (I personally am not very versed in the ways of accessibility)

Reply:
OK, I think I can think of some stuff, but I think we'll wait until we can create some more complex examples with Paste components to document it.